1.8/administration/installing/custom/cli.md

```diff
@@ -160,6 +160,11 @@ The DC/OS installation creates these folders:
     - 8.8.8.8
     ssh_port: 22
     ssh_user: <username>
+    use_proxy: 'true'
+    http_proxy: http://<your_http_proxy>/
+    https_proxy: https://<your_https_proxy>/
+    no_proxy: 
+    - '*.int.example.com' 
```

Comment:
I know that these are generic environment variables so we should either 1) state that and link them to the external documentation or 2) explain them here. cc @cmaloney